docs/participate/03_treasury_proposal.md

```diff
@@ -11,6 +11,46 @@ The first step in applying for a Treasury grant is either to join the community
 This will help you get valuable community feedback throughout the process.
 It also gives Council members an open and transparent way to measure community sentiment.
 
+When submitting for discussion, it is best practice to include a proposal document.
+[A reference document](https://docs.google.com/document/d/1DBb7jdgVVK09lwQPfOs6WFwbWUGKut7hMMmM_lEX0fA/edit) can be found in the KILT Protocol discussions this can be used as a template for future proposals.
+
+## DID Sign
+
+The proposed document must be DID signed.
+This requires a DID that must be associated with the proposal.
+Therefore, it requires the proposer to have a DID.
+The signature provides integrity and accountability about the submitter, which can give more confidence to the community and council.
+
+The DID Signature should be done via the [DIDSign](https://didsign.io/) dApp.
+The DIDSign doesn't have a database, nor store any data by the user.
+The following guide explains [how to create a signature using a DID with DIDSign](https://kilt-protocol.org/files/How-to-Guide-DIDsign.pdf).
+
+Once the document has been signed, the signature must be made available to others to verify.
+The easiest way to do this is to store it somewhere publicly accessible, e.g., on IPFS or as a GitHub gist.
+This must include the document and signature.
+The method of storing the document and the signature can vary, however, we recommend using IPFS or an equivelant such as Crust.
```

Comment:
```suggestion
Once the document has been signed, the signature must be made publicly available for verification.
Depending on the services and software used, accessing the storage may require different instructions, e.g., on IPFS or as a GitHub gist or cloud provider.
This must include the document and signature.
```

Reply:
@ntn-x2 What do you think? Has this cleared it up, or do you want to add anything else?

Reply:
> This must include the document and signature.
This is not required. The document itself is typically hosted somewhere else than the signature. So you would download it from Google Drive, while the signature could be stored also in Google Drive, or it could be stored on IPFS.